### PR TITLE
Add support for package:web v1.0.0

### PR DESCRIPTION
**Pull request by srujzs**
- Expands version range to >=0.5.1 <2.0.0.
- In order to do this, we add an `innerHtml` extension interop member both so we can support both major
versions and so we don't need to do manual conversions of JSAny.
- Also updates `createElement` calls to use the constructors supplied in package:web 0.5.1 onwards.
- Publishes version 8.0.12.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+## 8.0.13
```
Comment (srawlins) [inline CHANGELOG.md:1]: I actually haven't released 8.0.12 yet, so you can put these notes under 8.0.12

File: pubspec.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: dartdoc
-version: 8.0.12
+version: 8.0.13
```
Comment (srawlins) [inline pubspec.yaml:2]: Can just keep this as 8.0.12.

**Review by srawlins — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (srujzs) [reply CHANGELOG.md]: Done, thanks!

File: pubspec.yaml (hunk 2 above)
Comment (srujzs) [reply pubspec.yaml]: Done.

**Review by srawlins — APPROVED** [review]
Thanks a million, very cool!

**Pull request merged**